### DISCUSSION TITLE
Trying to add Azure.Identity to my main project

### DISCUSSION THREAD

**Discussion by cconner100** [unoplatform/uno#12098] [General]
Hi, i am trying to use AzureKeyvault in my Uno project.  I am trying to add the nuget package Azure.Identity but once i do it complains that in Windows the reference can not be found.  Its found in .net7.0 but not in windows.  Is this expected?
![Screenshot 2023-04-27 at 5 44 21 PM](https://user-images.githubusercontent.com/942585/234839767-a6625020-09e0-4d6c-983d-d4a0a8355ea9.png)

**Comment by jeromelaban** [comment]
This particular library is a netstandard2.0 library, it should be working fine. Are you getting errors in your build logs?

**Comment by cconner100** [comment]
4>C:\Program Files\dotnet\sdk\7.0.203\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(448,5): error NETSDK1082: There was no runtime pack for Microsoft.WindowsDesktop.App.WindowsForms available for the specified RuntimeIdentifier 'win10-arm'.
4>C:\Program Files\dotnet\sdk\7.0.203\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(448,5): error NETSDK1082: There was no runtime pack for Microsoft.WindowsDesktop.App.WindowsForms available for the specified RuntimeIdentifier 'win10-arm-aot'.

Funny these are ok in everything but windows runtime, ios and android work

**Reply by jeromelaban** [reply]
This looks like an environment issue. You may try to reinstall .NET and/or visual studio and see if it helps.

**Reply by cconner100** [reply]
OK thanks for the response, so i tried yet another approach.  I tried to just use the nuget package microsoft.data.sql.  I get the same issue

here is a repro of it
https://github.com/cconner100/UnoSqlData

All i am doing now is referencing the microsoft.data.sql nuget package and the only issue i get is an error when compiling windows
The error is
2>C:\Program Files\dotnet\sdk\7.0.300-preview.23179.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(448,5): error NETSDK1082: There was no runtime pack for Microsoft.WindowsDesktop.App.WindowsForms available for the specified RuntimeIdentifier 'win10-arm'.
2>C:\Program Files\dotnet\sdk\7.0.300-preview.23179.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(448,5): error NETSDK1082: There was no runtime pack for Microsoft.WindowsDesktop.App.WindowsForms available for the specified RuntimeIdentifier 'win10-arm-aot'.

i did uno-check --pre and everything is looking correct.  What am i doing incorrect

**Reply by jeromelaban** [reply]
Thanks. For some reason the WinUI head is not accepting that package. @nickrandolph have you ever seen something similar?

**Reply by cconner100** [reply]
Hey guys, this seems to happen to me with any db or azure standard windll.  strange that it works for ios and android but not windows.  This is a real show stopper for me

**Reply by muhdammar** [reply]
hi any solution regarding this?

**Reply by jeromelaban** [reply]
This is working fine with the latest bits from `Azure.Identity` (1.10.4) and .NET 8 with the latest uno project template. You'll need to add this specific workaround https://github.com/unoplatform/uno/issues/14555#issuecomment-1876175425, though to get full build of the `.Windows` head.
